<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="timeMenu">
    <property name="title">
     <string>Время</string>
    </property>
    <addaction name="setTimeAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="helpAction"/>
   </widget>
   <widget class="QMenu" name="mainMenu">
    <property name="title">
     <string>Главная</string>
    </property>
    <widget class="QMenu" name="bgMenu_2">
     <property name="title">
      <string>Фон</string>
     </property>
     <addaction name="BGchange1"/>
     <addaction name="BGchange2"/>
    </widget>
    <addaction name="bgMenu_2"/>
   </widget>
   <addaction name="timeMenu"/>
   <addaction name="helpMenu"/>
   <addaction name="mainMenu"/>
  </widget>
  <action name="setTimeAction">
   <property name="text">
    <string>Установить время</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="BGchange1">
   <property name="text">
    <string>Дали</string>
   </property>
  </action>
  <action name="BGchange2">
   <property name="text">
    <string>Второй</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
